dts/bindings/display/st,stm32-ltdc.yaml

```diff
@@ -32,9 +32,6 @@ properties:
   interrupts:
     required: true
 
-  pinctrl-0:
-    required: true
-
   hsync-pol:
```

Comment:
Rather than deleting, please add a description to explain when it's required and when it's not